components/StyledInput.js

```diff
@@ -38,6 +38,7 @@ const StyledInput = styled.input`
   box-sizing: border-box;
   outline: none;
   background-color: ${themeGet('colors.white.full')};
+  display:block;
```

Comment:
Should not be set globally, this would impact all the inputs that we have on the website

```suggestion
```